CODEOWNERS

```diff
@@ -1 +1,3 @@
-*       @navikt/team-esyfo
+* @navikt/team-esyfo
+package.json
```

Comment:
package i codeowners?

Reply:
Dette er et triks for at github ikke skal requeste review av team-esyfo for alle dependabot prs

Reply:
aha! Mulig å legge inn en kommentar om det, så det ikke ser ut som en copypaste feil? :P